[
    {
        "vuln": "CVE-XXXX-YYYY",
        "image": ".*",
        "tag": ".*",
        "reason": "An example"
    },
    {
        "vuln": "*",
        "image": "^docs.*",
        "tag": ".*",
        "reason": "The docs image is not deployed in production"
    },
    {
        "vuln": "pyup.io-38834 (CVE-2020-26137)",
        "image": ".*",
        "tag": ".*",
        "reason": "This is a Quay false fixable on ubi8:8.5"
    },
    {
        "vuln": "pyup.io-43975 (CVE-2021-33503)",
        "image": ".*",
        "tag": ".*",
        "reason": "This is a Quay false fixable on ubi8:8.5"
    }
]
